src/user/user.spec.js

```diff
@@ -54,4 +54,24 @@ describe("user :", () => {
       await expect(xo.createUser(simpleUser)).rejects.toMatchSnapshot();
     });
   });
+
+  describe(".delete() :", () => {
+    it("deletes a user successfully with id", async () => {
+      const userId = await xo.call("user.create", simpleUser);
+      expect(await xo.call("user.delete", { id: userId })).toBeTrue();
+      expect(find(await xo.getAllUsers(), { id: userId })).toBeUndefined();
+    });
+
+    it("fails trying to delete a user with a nonexistent user", async () => {
+      await expect(
+        xo.call("user.delete", { id: "nonexistentId" })
+      ).rejects.toMatchSnapshot();
```

Comment:
I don't like `rejects.toMatchSnapshot()`, it does not save the error properties in the snapshot, could you use `rejectionOf` with `toMatchSnapshot`?